test-packages/ts-ember-app/tsconfig.json

```diff
@@ -24,7 +24,7 @@
     "baseUrl": ".",
     "module": "es6",
     "experimentalDecorators": true,
-    "isolatedModules": true,
+    "isolatedModules": false,
```

Comment:
Why did this need to change?

Is this fallout from using yarn?

Reply:
Oh interesting, looking at the previous types errors, thin fixes an issue with the node types. 
Can you add a note / comment to check if isolatedModules can be flipped back to true when the dep updates?